.github/workflows/release.yml

```diff
@@ -26,6 +26,15 @@ jobs:
       - name: Install gox
         run: GO111MODULE=off go get github.com/mitchellh/gox
 
+      - name: "Set up QEMU"
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
```

Comment:
Do we really need this here? I don't see this line in [here](https://github.com/caarlos0/domain_exporter/blob/master/.github/workflows/build.yml#L41) for example. If it's not really needed then let's remove.

Reply:
we need this line for reason that this will init all platform qemu binaries, without this, we'll got a error on docker bulidx build ....

Reply:
Just found that it's `all` by default: https://github.com/docker/setup-qemu-action#inputs. Fine, let's leave it this way.